<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PTax 1.1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favicon.ico</normaloff>favicon.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow
{
background-color: rgb(100, 0, 0);
color: rgb(245, 245, 245);
border-color: rgb(0, 0, 0);}
</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel{
	color: rgb(220, 220, 220);
}
.QLineEdit:hover{
border: 2px solid gray;
}
.QLineEdit{
border-radius: 10px;
height: 20px;
border: 2px solid black;

}
QWidget{
background-color: rgb(100, 0, 0);
color: rgb(238, 238, 238);
border-color: rgb(0, 0, 0);}

.QPushButton
{
background-color: rgb(238, 238, 238);
color: rgb(42, 42, 42);
border: 4px solid gray;
width: 120px;
height: 22px;
border-radius: 8px;
}

.QPushButton:hover{
background-color: rgb(0, 0, 0);
color: rgb(238, 238, 238);
}

.QPushButton:pressed{
background-color: rgb(0, 0, 0);
color: rgb(238, 238, 238);
border: 4px solid white;
}
.QComboBox{
border-radius: 8px;
height: 20px;
border: 2px solid black;
}
.QComboBox:hover{
border: 2px solid gray;

}

</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="26" column="1">
     <widget class="QLineEdit" name="lineEditAfTax">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(16, 16, 16);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditZip">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(76, 76, 76);
border-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="lineEditCity">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(76, 76, 76);</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButtonLook">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Lookup</string>
      </property>
     </widget>
    </item>
    <item row="21" column="0">
     <widget class="QPushButton" name="pushButton_get_directions">
      <property name="styleSheet">
       <string notr="true">

</string>
      </property>
      <property name="text">
       <string>Get Directions</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QComboBox" name="comboBox_origin">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(76, 76, 76);</string>
      </property>
      <item>
       <property name="text">
        <string>Oakwood</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Townville</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="26" column="0">
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>98</width>
        <height>18</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>98</width>
        <height>18</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Price After Tax</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEditRate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(16, 16, 16);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>City</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="11" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Distance</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Street Address</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" rowspan="2">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Origin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignLeft">
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Job Address</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rate</string>
      </property>
     </widget>
    </item>
    <item row="25" column="1">
     <widget class="QLineEdit" name="lineEditBeTax">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(76, 76, 76);</string>
      </property>
     </widget>
    </item>
    <item row="21" column="1">
     <widget class="QPushButton" name="pushButton_dist">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Get Shipping Cost</string>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QLineEdit" name="lineEdit_Dist">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(16, 16, 16);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="20" column="1">
     <widget class="QLineEdit" name="lineEdit_shipping_cost">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(16, 16, 16);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditAddress">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(235, 235, 235);
color: rgb(76, 76, 76);</string>
      </property>
     </widget>
    </item>
    <item row="22" column="0" colspan="2">
     <widget class="Line" name="line_3">
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="25" column="0">
     <widget class="QLabel" name="label_6">
      <property name="maximumSize">
       <size>
        <width>108</width>
        <height>29</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Price before Tax</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditCounty">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(16, 16, 16);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>108</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>County</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Zip</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Shipping</string>
      </property>
     </widget>
    </item>
    <item row="20" column="0">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Shipping Cost</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_13">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tax Rate</string>
      </property>
     </widget>
    </item>
    <item row="23" column="0">
     <widget class="QLabel" name="label_14">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pricing</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 0, 0);
color: rgb(220, 220, 220);
background-color: rgb(220, 220, 220);
color: rgb(100, 0, 0);

</string>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menutest">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLicense"/>
   </widget>
   <addaction name="menutest"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
